source/vibe/web/validation.d

```diff
@@ -76,7 +76,7 @@ struct ValidUsername {
 		ret.nullify();
 
 		auto err = appender!string(); // TODO: avoid allocations when possible
-		if (validateUserName(str)) ret = ValidUsername(str);
+		if (validateUserName(str) != "") ret = ValidUsername(str);
```

Comment:
Good that these lines appear now in a diff. They are using the wrong overload. Should be:
`if (validateUserName(err, str)) ret = ValidUsername(str);`

I wouldn't have expected the DMD change to catch these kinds of bugs, too :)